common/src/main/java/com/example/android/uamp/media/MusicService.kt

```diff
@@ -419,6 +449,23 @@ open class MusicService : MediaLibraryService() {
             customCommand: SessionCommand,
             args: Bundle
         ): ListenableFuture<SessionResult> {
+            /*
+            Toggle spatial audio
+             */
+            if (customCommand.customAction == ACTION_TOGGLE_SPATIALIZATION) {
+                val enable = customCommand.customExtras.getBoolean(EXTRAS_TOGGLE_SPATIALIZATION)
+
+                if (enable)
```

Comment:
style nits:

- the `setSpatializationBehavior` lines are longer than 100 characters
- add braces since the if/else isn't one line